### ISSUE TITLE
Graylog failes silent if Elastic in Cluster Block Exception [FORBIDDEN/12/index read-only / allow delete (api)] and index gets rotated - messages are lost 

### ISSUE THREAD

**Issue by HenryTheSir** [Graylog2/graylog2-server#9955]
Graylog failes silent in GUI and drops messages if elastic is in [FORBIDDEN/12/index read-only / allow delete (api)] and the current index gets cycled 

## Expected Behavior
Fail with message in Graylog GUI, Log messages as dropped or do not drop messages. 

## Current Behavior
GUI gives the feedback, as if everything works well, In/Out messages are displayed, in Journal no high amound of unprocessed messages present. 
In System Overview no failed index attempts in last 24h get displayed, but no messages are indexed. 
Stacktraces are thrown in logfile:
```
Couldn't point deflector to a new index
org.graylog2.indexer.ElasticsearchException: Couldn't remove alias log_deflector from indices [log_28]

blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];
	at org.graylog2.indexer.cluster.jest.JestUtils.specificException(JestUtils.java:110) ~[graylog.jar:?]
	at org.graylog2.indexer.cluster.jest.JestUtils.execute(JestUtils.java:60) ~[graylog.jar:?]
	at org.graylog2.indexer.cluster.jest.JestUtils.execute(JestUtils.java:65) ~[graylog.jar:?]
	at org.graylog2.indexer.indices.Indices.removeAliases(Indices.java:661) ~[graylog.jar:?]
	at org.graylog2.indexer.MongoIndexSet.cleanupAliases(MongoIndexSet.java:352) ~[graylog.jar:?]
	at org.graylog2.periodical.IndexRotationThread.checkAndRepair(IndexRotationThread.java:149) ~[graylog.jar:?]
	at org.graylog2.periodical.IndexRotationThread.lambda$doRun$0(IndexRotationThread.java:76) ~[graylog.jar:?]
	at java.lang.Iterable.forEach(Iterable.java:75) [?:?]
	at org.graylog2.periodical.IndexRotationThread.doRun(IndexRotationThread.java:73) [graylog.jar:?]
	at org.graylog2.plugin.periodical.Periodical.run(Periodical.java:77) [graylog.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

```
Couldn't get active write index
org.graylog2.indexer.indices.TooManyAliasesException: More than one index in deflector alias: [log_76, log_28]
	at org.graylog2.indexer.indices.Indices.aliasTarget(Indices.java:355) ~[graylog.jar:?]
	at org.graylog2.indexer.MongoIndexSet.getActiveWriteIndex(MongoIndexSet.java:204) ~[graylog.jar:?]
	at org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical.lambda$schedule$4(IndexFieldTypePollerPeriodical.java:201) ~[graylog.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

[log_76, log_28] difference in number comes from cycling after time and not being alerted by this error, so that we have quite a time till someone missed their loglines. 

## Possible Solution
Keep messages in internal journal, and give system message in GUI that things are messed up. 

## Steps to Reproduce (for bugs)

1. Set current write index to read only/ cluster block 
2. Rotate active index in Graylog GUI/Rest API
3. Described issue should occure
4.

## Context
Messages are lost silently and systems behaves as if everything works fine. 

## Your Environment

* Graylog Version: 3.3
* Java Version: 11
* Elasticsearch Version: 6.8
* MongoDB Version: 4.2
* Operating System: RHEL
* Browser version: doesn't matter

**Comment by dennisoelkers** [comment]
Hey @HenryTheSir,

thanks a lot for catching and describing this nasty issue. I can reproduce it and have prepared PRs fixing the underlying issue. I hope to get them in on time for the next patch releases, but a change like this needs a thorough review. I will keep you updated!

**Comment by dennisoelkers** [comment]
Hey @HenryTheSir,

the issue should be resolved in `3.3.10` and `4.0.2`. Thanks again for the report!

**Comment by HenryTheSir** [comment]
Hey @dennisoelkers,

that was fast fixing. Thank you!
